recipes/cerberus/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.9.2" %}
+{% set sha256 = "b122c7b2cbf856ea2587e187fac968fc8dcd49d47aa1f239abd9eaa0ed86a7ce" %}
+
+package:
+  name: cerberus
+  version: {{ version }}
+
+source:
+  fn: Cerberus-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/c1/7a/65f3aa48279cda81208ccca4c932e63fedaf02f80f1fb6a482a7b8d8f239/Cerberus-0.9.2.tar.gz
```

Comment:
Please use the [templated url]( https://github.com/conda-forge/staged-recipes/blob/3080b8948ce291964824b04ad01e5b68186af404/recipes/example/meta.yaml#L18 ) from the example.